stan/math/prim/scal/fun/sum.hpp

```diff
@@ -0,0 +1,22 @@
+#ifndef STAN_MATH_PRIM_SCAL_FUN_SUM_HPP
+#define STAN_MATH_PRIM_SCAL_FUN_SUM_HPP
+
+namespace stan {
+namespace math {
+
+/**
+ * Calculates sum of values given as input. For scalar inputs this is identity
+ * function.
+ *
+ * @tparam T Type of element.
+ * @param v Specified value.
+ * @return Same value (the sum of one value).
+ */
+template <typename T>
+inline T sum(const T& v) {
```

Comment:
Return type should be `const T&` for same reasons as before.

Doc should also doc this version of the function.  It's just an overload, not a specialization.